docs/self-managed/components/modeler/web-modeler/overview.md

```diff
@@ -2,7 +2,30 @@
 id: overview
 title: "Web Modeler on Self-Managed"
 sidebar_label: "Overview"
```

Comment:
Are we able to set this up in the sidebar so when someone clicks "Web Modeler" it automatically takes them to the overview page, rather than a separate sidebar item for the overview?